components/devtools/actors/timeline.rs

```diff
@@ -214,7 +214,7 @@ impl Actor for TimelineActor {
                 **self.is_recording.lock().as_mut().unwrap() = true;
 
                 let (tx, rx) = channel::<TimelineMarker>();
-                self.script_sender.send(SetTimelineMarkers(self.pipeline, self.marker_types.clone(), tx));
+                let _ = self.script_sender.send(SetTimelineMarkers(self.pipeline, self.marker_types.clone(), tx));
```

Comment:
Unless there is a specific reason to ignore errors, use `.unwrap()` on the `Result` instead.